topics/admin/tutorials/cvmfs/tutorial.md

```diff
@@ -227,6 +253,24 @@ Now all we need to do is tell Galaxy how to find it! This tutorial assumes that
 >
 {: .hands_on}
 
+# Common Production Questions
+
+> ### {% icon question %} For the most used datasets (for ex. hg38) could we have a local copy, or would that be irrelevant?
+> This would be irrelevant, the most used datasets will stay in the cache. CVMFS uses an LRU or "Least Recently Used" cachec (see their [docs](https://cvmfs.readthedocs.io/en/latest/cpt-details.html#disk-cache)), so whenever it runs out of space, it will remove the least recently used file. If you have a file that is very commonly used, it will remain in the cache.
```

Comment:
```suggestion
> This would be irrelevant, the most used datasets will stay in the cache. CVMFS uses a Least Recently Used (LRU) cache (see their [docs](https://cvmfs.readthedocs.io/en/latest/cpt-details.html#disk-cache)), so whenever it runs out of space, it will remove the least recently used file. If you have a file that is very commonly used, it will remain in the cache.
```